src/runner/runESLint.js

```diff
@@ -122,6 +122,15 @@ const getComputedFixValue = ({ fix, quiet, fixDryRun }) => {
   return undefined;
 };
 
+const removeUndefinedFromObject = object => {
+  return Object.keys(object).forEach(v => {
+    if (object[v] === undefined) {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
The code using Object.assign timed out the test.
Please comment if you have any other good code.